### DISCUSSION TITLE
开启vpn状态下Windows 聚焦无法自动更新

### DISCUSSION THREAD

**Discussion by veyyyyyyy** [v2fly/v2ray-core#2582] [Q&A]
系统是win11,v2ray版本是5.23,绕过大陆，自动配置系统代理。

这种状态下，Windows聚焦，就是电脑开机锁屏壁纸，无法自动更新，关闭v2ray才恢复正常，求解，谢谢~

**Comment by PandaRyshan** [comment]
听起来像是 Win 的 [Loopback 问题](https://learn.microsoft.com/en-us/windows/iot-core/develop-your-app/loopback)，需要使用 [AppContainer Loopback Exemption Utility](https://docs.telerik.com/fiddler/configure-fiddler/tasks/configurefiddlerforwin8) 这个工具去解决 Loopback 的问题，[下载链接](https://telerik-fiddler.s3.amazonaws.com/fiddler/addons/enableloopbackutility.exe)。

Clash for Windows 和 V2rayN 都自带了这个工具，如果你使用了其中一个工具，试着升级到最新版，并找到这个工具。全选所有 App，然后 Save Changes。这样应该就能解决了。
